cypress/e2e/settings/set-integration-manager.spec.ts

```diff
@@ -0,0 +1,64 @@
+/*
+Copyright 2023 Suguru Hirahara
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+/// <reference types="cypress" />
+
+import { HomeserverInstance } from "../../plugins/utils/homeserver";
+
+const USER_NAME = "Alice";
+
+describe("Set integration manager on General settings tab", () => {
+    let homeserver: HomeserverInstance;
+
+    beforeEach(() => {
+        cy.startHomeserver("default").then((data) => {
+            homeserver = data;
+            cy.initTestUser(homeserver, USER_NAME);
+        });
+    });
+
+    afterEach(() => {
+        cy.stopHomeserver(homeserver);
+    });
+
+    it("should set integration manager", () => {
+        cy.openUserSettings("General");
+        cy.contains("Manage integrations").should("exist");
+
+        // Make sure integration manager's toggle switch is enabled
+        cy.get(".mx_SetIntegrationManager .mx_ToggleSwitch_enabled").should("exist");
+
+        // Check the default margin set to mx_SettingsTab_heading on the settings tab
+        cy.get(".mx_SettingsTab > .mx_SettingsTab_heading").should("have.css", "margin-right", "100px");
```

Comment:
These type of assertions on css are brittle and not in the spirit of end to end testing. E2E tests should use assertions that mimic what a *user* would expect.
The percy snapshot is enough here.

Reply:
> I may be wrong, but does Percy run every time when a commit is merged to develop branch, right?

I believe that's correct.

> I am a little bit worried that running Percy less frequently (https://github.com/vector-im/element-web/issues/24731) could increase the chance that a visual regression would slip in by chance. @Johennes I would like to ask your opinion about how much a test should depend on Percy. An official policy about it would be helpful, if any.

This is also correct and while it's not exactly ideal, it's the best we're able to do with our current budget. Right now we hit our Percy quota half-way through each monthly cycle which means for the second half of each month, we run no Percy tests at all.

Reply:
@kerryarchibald Would you mind letting me know your thought about whether the assertion should be removed or not? Since it looks that the style rules under `mx_SetIntegrationManager` fortunately have been defined strictly, the risk of a regression should have been minimized to the level that we would not need the assertion..

Reply:
@kerryarchibald it looks like this is waiting on your response, I think.

Reply:
I don't think e2e testing css values is a sustainable solution to our brittle css. I still think the assertion should be removed, but I will approve the PR anyway.

Reply:
OK, let's leave it in and if it causes problems, remove it eagerly.